modules/widgets/system-tray.nix

```
@@ -214,17 +216,9 @@ in
             iconSpacing = icons.spacing;
           };
         };
-        mergedSettings = lib.recursiveUpdate sets settings;
-      in
-      {
-        name = "org.kde.plasma.systemtray";
         extraConfig = ''
           (widget) => {
-            const tray = desktopById(widget.readConfig("SystrayContainmentId"));
-            if (!tray) return; // if somehow the containment doesn't exist
-
-            ${widgets.lib.setWidgetSettings "tray" mergedSettings}
-            ${widgets.lib.addWidgetStmts "tray" "trayWidgets" items.configs}
+            ${widgets.lib.addWidgetStmts "widget" "tray_widgets" items.configs}
```

```suggestion
            ${widgets.lib.addWidgetStmts "widget" "trayWidgets" items.configs}
```

Nit, just to keep things consistent

---

Thanks. Fixed